lib/galaxy/tool_util/deps/mulled/mulled_build.py

```diff
@@ -513,7 +513,7 @@ def add_single_image_arguments(parser):
     )
 
 
-def target_str_to_targets(targets_raw):
+def parse_targets(targets_raw):
```

Comment:
Not sure the renaming is worth the risk of breaking downstream code using the `galaxy-tool-util` package?

Reply:
I agree. This should - if at all - be done in steps with a deprecation warning announcing the change.

So why did I even do it? Its functionality - or rather the pieces that make up its functionality - are not reused as widely as it could be. When you run `git grep -n '.split(' *.py` in the `mulled` folder, you'll see that the parsing is reinvented over and over again. And I am hesitant to add another version of the paring of container definitions in a new linting module. So it the intention is not just to rename this function, but to decompose it into reusable parts and reuse them whenever a string is parsed that defines a mulled container.